infra/package.json

```diff
@@ -11,19 +11,20 @@
     "cdk": "cdk"
   },
   "peerDependencies": {
-    "constructs": "^10.0.0",
-    "aws-cdk-lib": "^2.0.0"
+    "aws-cdk-lib": "2.60.0",
+    "constructs": "^10.0.0"
   },
   "devDependencies": {
     "@types/jest": "^26.0.10",
     "@types/node": "10.17.27",
-    "aws-cdk-lib": "^2.0.0",
+    "aws-cdk-lib": "^2.60.0",
```

Comment:
IMO we should fix the version of `aws-cdk-lib` instead of using this ">=" style of dependency specification

Reply:
I agree, but we are restricted by `eks-blueprints` (see above)